### ISSUE TITLE
Update cake build, see if it's still behaving nicely with AppVeyor

### ISSUE THREAD

**Issue by maldworth** [acarteas/FileCache#27]
Assign this issue to me. I will take a look when I have some time

**Comment by acarteas** [comment]
Looks like unit tests are hanging on appveyor now?

**Comment by maldworth** [comment]
So I re-ran the original commit I made, multiple times and it passes every time.
https://ci.appveyor.com/project/maldworth/filecache

So it might be possible that one of the last 3 commits started to fail.
https://ci.appveyor.com/project/acarteas/filecache/history

Looking at the history, #33 looks like some of those tests caused this. So me recommendation is to maybe ask the original contributor for the PR to look at the tests they introduced

**Comment by maldworth** [comment]
usually I don't accept PR's unless the build passes

**Comment by acarteas** [comment]
I figured you knew what you were doing since you were the one who
originally set up the appveyor code :/

**Comment by acarteas** [comment]
Sorry, didn't read your full thread.  All tests pass locally.  AppVeyor just seems to hang during the testing phase.

**Comment by CelerityChip** [comment]
Something with parallel test execution?  I'm actually able to make the prior build hang locally by running tests in parallel with a .runsettings file like so:

<?xml version="1.0" encoding="utf-8"?>
<RunSettings>
  
  <MSTest>
    <Parallelize>
      <Workers>2</Workers>
      <Scope>ClassLevel</Scope>
    </Parallelize>
  </MSTest>
</RunSettings>

**Comment by acarteas** [comment]
I don't see anything like that in the Cake build.  Reopening this as a reminder until it gets sorted out.

**Comment by acarteas** [comment]
closing as I will be documenting future work on #45
